gcloud-java-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBlobSnippets.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.example.storage.snippets;
+package com.google.cloud.examples.storage.snippets;
```

Comment:
The repo uses this model, we should continue.  In general, for Java, we are following com.example.something model, so that if someone grabs our source file to start their own project, they are more likely to change it, than having a com.google on their own code.  What you are doing is fine.